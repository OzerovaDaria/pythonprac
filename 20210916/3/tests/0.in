[31, 18, 33, 7, 78, 68, 90, 74, 82, 21]