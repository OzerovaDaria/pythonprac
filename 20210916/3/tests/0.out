[31, 18, 33, 7, 78, 68, 90, 74, 82, 21, 67, 32, 99, 64, 81, 3, 78, 22, 42]
